### DISCUSSION TITLE
dialog box for saving file

### DISCUSSION THREAD

**Discussion by gamemaJ815** [yasirkula/UnityNativeGallery#291] [Q&A]
dose saving features have a dialog box? i  tryto make an app with your asset to save a picture that i take a screen shot for and set path  using dialog box, however it only saved and dose not show any kind of dialog box is this feature does not allow in this asset?

**Comment by yasirkula** [comment]
No dialog box is shown. Image is saved to the specified album (iOS14+ not supported) with the specified filename. You can use NativeFilePicker.ExportFile to export files to desired locations.

**Reply by gamemaJ815** [reply]
thanks for fast reply i try it out =)
